README.md

```diff
@@ -64,3 +64,37 @@ Use with `-k` for keycloak authentication. Credentials are read from the `.netrc
 Use `-d` to increase verbosity. Specify multiple times to increase more.
 
 Custom filters could be defined in `filters.txt` file.
+
+# GSS Admin API Client
+To get full parameter list run
+```
+source bin/activate
```

Comment:
also
`virtualenv .`
and
`pip install -r requirements.txt`

are also required, right? maybe we can move it from the separate scripts' sections to the top overview.